app/controllers/unsubscribe_feedbacks_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UnsubscribeFeedbacksController < ApplicationController
+  def new
+    subscription = Subscription.find(subscription_id)
+    @subscription = SubscriptionPresenter.new(subscription)
+    @unsubscribe_feedback_form = UnsubscribeFeedbackForm.new
+  end
+
+  def create
+    @subscription = Subscription.find(subscription_id)
+    @unsubscribe_feedback_form = UnsubscribeFeedbackForm.new(unsubscribe_feedback_params)
+
+    if @unsubscribe_feedback_form.valid? && @subscription.unsubscribe_feedbacks.create(unsubscribe_feedback_params)
+      if current_jobseeker
+        redirect_to jobseekers_subscriptions_path, success: t(".success")
+      else
+        render :confirmation
+      end
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def subscription_id
+    ParameterSanitiser.call(params).require(:subscription_id)
+  end
+
+  def unsubscribe_feedback_params
+    ParameterSanitiser.call(params.require(:unsubscribe_feedback_form).permit(:reason, :other_reason, :additional_info))
```

Comment:
Do we want to decide a convention for which way round we do the sanitisation? (i.e `require.permit` inside here vs outside on line 26)

Reply:
Hmm, maybe! Also part of me thinks we have this parameter sanitisation in enough places that there must be a nicer way of doing it  😕

Reply:
Have changed it to be consistent in this file for now.